README.md

```diff
@@ -67,6 +69,8 @@ This tap:
    can be found. For example, it might look like:
    `https://mycompany.atlassian.net`.
 
+   The `groups` specifies groups for users stream. `groups` is optional parameter. Default value of groups is `["jira-administrators", "jira-software-users", "jira-core-users", "jira-users", "users"]`.
```

Comment:
```suggestion
   The `groups` specifies groups for users stream. It is an optional parameter. Default value is `["jira-administrators", "jira-software-users", "jira-core-users", "jira-users", "users"]`.
```

Reply:
Yeah, it looks good, I have updated the comment as per suggestion.